packages/app/routes.tsx

```diff
@@ -42,6 +45,15 @@ const routes = (
       <Route path="my-account" component={Todo} />
       <Route path="my-skills" component={Todo} />
     </Route>
+
```

Comment:
Please merge dev onto your branch and follow the standard I have set for the API demo routes

Reply:
Merged dev into my feature branch, and formatted routes to go under api as well.
:+1: